migrations/etl/1_Mine_profile_detail_location.sql

```diff
@@ -149,8 +149,7 @@ BEGIN
     INSERT INTO mine_location(
         mine_location_guid  ,
         mine_guid           ,
-        latitude            ,
-        longitude           ,
+        geom                ,
```

Comment:
I was assuming an 'expand and then contract'.. so we'd keep mine_location.latitude/longitude until we meed to contract.  This needs to be timed with the other MDS methods that currently work with textual lat/long (e.g. the map rendering of points); if that map still needs textual fields, we can get rid of the two columns but use the PostGIS functions (e.g. `ST_X(geom) as longitude`, `ST_Y(geom) as latitude`)
```

Reply:
Good catch.  Done